Automattic/simplenote-macos
# Privacy Notice for California Users

### Fix
In this (quick!) PR we're adding a new Privacy Notice for California Users.

@aerych you game for a quick PR sir?
Thank yooou!!

### Test
1. Launch the App
2. Click `Simplenote` > `About Simplenote`

- [x] Verify there's a new link at the bottom that reads `Privacy Notice for California Users`
- [x] Verify that clicking over such link opens [this URL](https://automattic.com/privacy/#california-consumer-privacy-act-ccpa)

### Release
`RELEASE-NOTES.txt` was updated in dd0d46ec56da5194f87de18c5682a08e6e7b4388 with:
 
> New Privacy Notice for California Users

### Screenshots
<img width="459" alt="Screen Shot 2020-06-29 at 6 36 04 PM" src="https://user-images.githubusercontent.com/1195260/86058515-6b09e680-ba37-11ea-99c6-04ff2ad4a3d0.png">
